### PR TITLE
Options request two

### PR DESCRIPTION
**Pull request by grace000**
Allow additional headers of PUT, POST for second options request.

### CODE REVIEW

File: src/main/java/github/grace000/erversay/RequestRouter.java [hunk 1]
```diff
@@ -1,48 +1,70 @@
...
+            if(isStandardOptionsRequest(method)) {
+                getResponseForOptions();
+            }
+        } else if(path.equals(METHODS_TWO_URI)) {
+            if(isStandardOptionsRequest(method) || method.equals("PUT") || method.equals("POST")) {
+                getResponseForOptionsTwo();
+            }
+        } else {
+            getResponseForNotFound();
         }
```
Comment (BWoodfork) [inline src/main/java/github/grace000/erversay/RequestRouter.java:36]: This `if` conditional is getting pretty gnarly. What's your plan to clean it up?

Comment (grace000) [reply src/main/java/github/grace000/erversay/RequestRouter.java:36]: This is where I was going to use a switch statement based on the methods instead, and then cycle through and match up the incoming path.

Comment (BWoodfork) [reply src/main/java/github/grace000/erversay/RequestRouter.java:36]: You gonna do it as part of this PR?

Comment (grace000) [reply src/main/java/github/grace000/erversay/RequestRouter.java:36]: No, I'd prefer to do it as part of the POST PR.

Comment (BWoodfork) [reply src/main/java/github/grace000/erversay/RequestRouter.java:36]: Why?

Comment (grace000) [reply src/main/java/github/grace000/erversay/RequestRouter.java:36]: I should be on the post request this week. I can either add a card to refactor that portion of the options request or address it within the post request PR. I'd like to maintain the current sprint schedule.

**Pull request merged**